scripts/publish-release.js

```diff
@@ -88,8 +88,11 @@ const cwd = process.cwd()
 
   await Promise.allSettled(
     packageDirs.map(async (packageDir) => {
-      const pkgJson = await readJson(
-        path.join(packagesDir, packageDir, 'package.json')
+      const pkgJson = JSON.parse(
+        await fs.promises.readFile(
+          path.join(packagesDir, packageDir, 'package.json'),
+          { encoding: 'utf-8' }
```

Comment:
```suggestion
          'utf-8' 
```

Lets keep these options consistent with the other usages

Reply:
Changed in [`051c6ed` (#56917)](https://github.com/vercel/next.js/pull/56917/commits/051c6ed13dfaef686e5cff03acbc09b6f4de6ed3).